### ISSUE TITLE
Visible label and accessible name do not match - SiteImprove - Registrar Credential Validation

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9043]
[https://my2.siteimprove.com/Accessibility/995867/NextGen/Issue/1?ruleId=14&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmy2.siteimprove.com%2FAccessibility%2F995867%2FNextGen%2FIssue%2F1%3FruleId%3D14%26pageSegments%3D%26issueKind%3D1%26exceptTags%3D1%2C2%26conformance%3D%26siteTargetIssueKinds%3D1%2C2%26wcagVersion%3D21%26siteTarget.wcagVersion%3D21%26siteTarget.conformanceLevels%3D0%2C1%2C3%2C4%26siteTarget.issueKinds%3D1%2C2&data=05%7C02%7Cbrian-dickey%40uiowa.edu%7Ca13bda7b6932452f63ef08ddc56afbe7%7C1bc445959aba4fc3b8ec7b94a5586fdc%7C1%7C0%7C638883785462453933%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=PNDE6NQVtReyyFFrHbXquo48uKEZUk5kjHP888HsDgE%3D&reserved=0)

**Comment by briand44** [comment]
I reached out to Josh to see if he wants to touch base with CeCredential Validation folks or if we can make changes to this.

**Comment by joewhitsitt** [comment]
Sent a response with the above screenshot:

>I think we wanted to check with you previously to make sure the markup was still compliant with any agreements you have with the vendor. If there is some leeway here, I think the quick fix here is to remove the aria-label attributes (not visible to sighted users) that decorate the link.

>The more involved fix would be to replace the aria-label and text associated with the link to https://www.chea.org/search-institutions as "CHEA.org Search Institutions" or something but we would want to make sure they approve of that change.
